package.json

```diff
@@ -49,8 +49,10 @@
     "core-js": "3.8.1",
     "email-validator": "2.0.4",
     "formik": "2.2.6",
+    "i": "^0.3.7",
```

Comment:
What is this? I looks like you may have done `npm i i` which installed the package `i` lol

Reply:
unless that was intentional